### PR TITLE
Update data-lake-analytics.md

### PR DESCRIPTION
**Pull request by burkeholland**

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/data-lake-analytics.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-29/docs-ref-services/data-lake-analytics.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C4%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708041855301133-195%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/data-lake-analytics.md [hunk 1]
```diff
@@ -18,17 +18,37 @@ ms.service: Data Lake Analytics
...
 
-### Management
 ```bash
 npm install azure-arm-datalake-analytics
 ```
 
-##Example
+### Example
+
+This example lists all of the data analytics accounts for a given subscription.
```
Comment (johnpapa) [inline docs-ref-services/data-lake-analytics.md]: "data" or "data lake"?

Comment (burkeholland) [inline docs-ref-services/data-lake-analytics.md]: nice catch - should just be analytics

**Comment by burkeholland** [comment]
#sign-off per @TomArcher

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/data-lake-analytics.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-29/docs-ref-services/data-lake-analytics.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C7%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708072010372882-195%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**